packages/endpoint/0.13.0/manifest.yml

```diff
@@ -5,7 +5,7 @@ description: Protect your hosts with threat prevention, detection, and deep secu
 version: 0.13.0
 categories: ["security"]
 # Options are experimental, beta, ga
-release: beta
+release: experimental
```

Comment:
@jonathan-buttner Ups, find replace. I assume this should stay beta, correct?

Reply:
Yeah I think product wanted it to stay as `beta`.